frontend/src/api/open-hands-axios.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
```

Comment:
can we delete request.ts now?

Reply:
The old requests function was under `services/api.ts` which is also deleted in this PR